addons/forum-search/userscript.js

```diff
@@ -227,7 +227,10 @@ function appendSearch(box, query, page, term, msg) {
 }
 
 export default async function ({ addon, global, console, msg }) {
-  await addon.tab.loadScript(addon.self.lib + "/thirdparty/cs/scratchblocks-v3.5.2-min.js");
+  if (!window.scratchblocks3Enabled) {
+    await addon.tab.loadScript(addon.self.lib + "/thirdparty/cs/scratchblocks-v3.5.2-min.js");
+  }
+
```

Comment:
Does this mean forum-search already used scratchblocks 3 before this? I didn't know!

Reply:
If so, why didn't it load the translations as well?

Reply:
Translations arent supported in better quoter?

Reply:
But this is forum search?

Reply:
> But this is forum search?

i make too many typos